packages/host/src/lib/host.ts

```diff
@@ -352,12 +352,12 @@ export class Host implements IComponent {
             ({ service: this.service, apiVersion: this.apiVersion, version, build: this.build }));
 
         this.api.get(`${this.apiBase}/config`, () => this.publicConfig);
-
+        this.api.get(`${this.apiBase}/status`, () => this.getStatus());
         this.api.get(`${this.apiBase}/topics`, () => this.serviceDiscovery.getTopics());
         this.api.use(this.topicsBase, (req, res, next) => this.topicsMiddleware(req, res, next));
 
         this.api.upstream(`${this.apiBase}/log`, () => this.commonLogsPipe.getOut());
-
+        this.api.duplex(`${this.apiBase}/platform`, (stream, headers) => this.cpmConnector?.handleCommunicationRequest(stream, headers));
```

Comment:
Does that mean we can call the platform from STH?

Reply:
Ok, as explained, this is for handling a communication stream between hub and platform. Resolving this question.